### DISCUSSION TITLE
How are cutscenes added?

### DISCUSSION THREAD

**Discussion by TheWooomy** [KadeArchive/Kade-Engine#1192] [Q&A]
a few months ago i added a cutscene using a tutorial made by brightfyre to add cutscenes. Now kade engine added a cutscenes feature but i don't know how to use it. Someone can pls teach me how to add a cutscene

**Comment by isakube** [comment]
webm, ogg, mp4. i have no idea about anything else

**Reply by TheWooomy** [reply]
i need to add a webm video

**Comment by ghost** [comment]
TheWooomy, try using this tutorial: https://www.youtube.com/watch?v=k67RvU6qI_Q

It's really helpful :D

**Reply by TheWooomy** [reply]
I've used that tutorial but when the cutscenes ends the game crashes lol

**Reply by ghost** [reply]
Hmmm, I don't know if you have followed it carefully. If you did, then something is wrong with the files of your song.
